tests/backends/test_features.py

```diff
@@ -1,8 +1,23 @@
+from unittest import skipUnless
+
 from django.db import connection
-from django.test import TestCase
+from django.test import TransactionTestCase, mock
+
 
+class TestDatabaseFeatures(TransactionTestCase):
 
-class TestDatabaseFeatures(TestCase):
+    available_apps = ['backends']
 
     def test_nonexistent_feature(self):
         self.assertFalse(hasattr(connection.features, 'nonexistent'))
+
+    @skipUnless(connection.vendor == 'mysql', "This is a mysql-specific issue")
+    def test_mysql_supports_transactions(self):
+        """
+        Test that mySQL backend supports transactions.
```

Comment:
Docstring equivalent to test method name doesn't add much value. For future reference, we state the expected behavior rather than "Test that..." prefix, since all tests test things.